app/templates/footer.html

```diff
@@ -0,0 +1,10 @@
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" rel="stylesheet">
+<footer>
+    <p class="footer">
+        Whoogle Search v{{ version_number }} ||
+        <a id="gh-link" href="https://github.com/benbusby/whoogle-search">{{ translation['github-link'] }}</a>
+        {% if newest_version != version_number %}
```

Comment:
I think this could be improved a bit to allow users using branch builds with potentially a higher version number (i.e. branch build is `v0.6.2`, latest release is `v0.6.1`) to not trigger this check.

Off the top of my head, you could convert the version to an int easily enough with something like:

```py
int(''.join(filter(str.isdigit, version_str)))
```

and then check if the latest version is actually higher than the current version, but you should probably perform that logic outside of the template and then pass in the result as a template var.